Backpack/Card/Classes/BPKDividedCard.m

```diff
@@ -0,0 +1,121 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKDividedCard.h"
+#import "BPKCardDivider.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Color.h>
+#import <Backpack/Radii.h>
+#import <Backpack/Shadow.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKDividedCard()
+@property ( strong, nonatomic ) UIStackView *contentView;
+@property ( strong, nonatomic ) BPKCardDivider *lineView;
+@property ( assign, nonatomic ) UIView *primarySubview;
+@property ( assign, nonatomic ) UIView *secondarySubview;
+@end
+
+
+@implementation BPKDividedCard
+
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithPadded:(BOOL)padded {
+    self = [super initWithPadded:padded];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithSubviews:(UIView*)primarySubview withSecondarySubview:(UIView*)secondarySubview {
+    self = [super initWithPadded:true];
+    
+    _primarySubview = primarySubview;
+    _secondarySubview = secondarySubview;
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (void)setPadded:(BOOL)padded {
+    [super setPadded:padded];
+    self.contentView.spacing = padded ? BPKSpacingBase : 0;
+}
+
+- (void)setOrientation:(UILayoutConstraintAxis)orientation {
+    _orientation = orientation;
+    self.lineView.orientation = orientation;
+    self.contentView.axis = orientation;
+}
+
+- (void)setSubviews:(UIView*)primarySubview withSecondarySubview:(UIView*)secondarySubview {
+    [self.contentView removeArrangedSubview:self.primarySubview];
+    [self.contentView removeArrangedSubview:self.secondarySubview];
+
+    self.primarySubview = primarySubview;
+    self.secondarySubview = secondarySubview;
+    
+    [self.contentView insertArrangedSubview:self.primarySubview atIndex:0];
+    [self.contentView insertArrangedSubview:self.secondarySubview atIndex:2];
+}
+
+#pragma mark - Private
+
+- (void)setupViews {
+    self.contentView = [[UIStackView alloc]initWithFrame:CGRectZero];
+    [self.contentView setUserInteractionEnabled:false];
+    self.contentView.distribution = UIStackViewDistributionFillProportionally;
```

Comment:
Shouldn't we have `self.contentView.alignment = UIStackViewAlignmentFill` here too?

Reply:
I'm not sure tbh. I guess it probably better than not though, given that the consumer is then free to utilise the whole space available to each subview